src/modules/LanItems.rb

```diff
@@ -1812,14 +1812,14 @@ def setup_basic_device_options(devmap)
     # @param devmap [Hash] hash of a device's sysconfig variables
     # @param slaves [array] list of strings, each string is a bond slave name
     #
-    # @return [Hash] updated device map
+    # @return [Hash] a copy of updated device map
```

Comment:
well, it is still not clear that it do not modify original hash...so maybe `updated copy of device map`?